src/core/document.js

```diff
@@ -849,6 +850,51 @@ class Page {
     );
     return shadow(this, "jsActions", actions);
   }
+
+  async collectAnnotationByType(
```

Comment:
Shouldn't this be `collectAnnotationsByType`, in plural, given that we collect multiple (all) of them?